src/components/PickerMonth.vue

```diff
@@ -52,17 +52,17 @@
       </button>
     </header>
     <div
-      role="grid"
       aria-labelledby="year-button"
     >
       <button
         v-for="month in months"
         :key="month.timestamp"
-        type="button"
+        :aria-selected="month.isSelected"
```

Comment:
I would add `aria-disabled` too in the buttons that can be disabled, WDYT?

Reply:
Done!